README.md

```diff
@@ -157,7 +157,11 @@ In order to use the API within `polkadot.js` you'll need to configure the follow
     "_enum": ["Disabled", "Enabled"]
   },
   "Restriction": {
-    "_enum": ["None", "SenderOwnsAllInputs"]
+    "_enum": {
+      "None": null,
+      "SenderOwnsAllInputs": null,
+      "FixedNumberOfInputs": "num_inputs<u32>"
```

Comment:
I may be wrong but I think you'll need a new top level type. For example it could read:

```json
  "Restriction": {
    "_enum": ["None", "SenderOwnsAllInputs"]
    "_enum": {
      "None": null,
      "SenderOwnsAllInputs": null,
      "FixedNumberOfInputs": "FixedNumberOfInputsRestriction"
  },
  "FixedNumberOfInputsRestriction": {
    "num_inputs": "u32"
  }
```

I may be wrong about that though (docs arent overly clear). The way to test is to go to https://polkadot.js.org/apps/  configure it to your node and put in the types under the develop tab. We can look at this tomorrow if you like and work out the rigth answer